src/pages/crafts/index.js

```diff
@@ -119,7 +119,7 @@ function Crafts() {
                     tooltipContent={
                         <div>
                             {t(
-                                "Sets all fuel prices to 0, as that's more or less the case if you use fuel to power your hideout",
+                                "Sets fuel canister cost to 0 for crafts requiring fuel canisters when using empty fuel canisters from generator.",
```

Comment:
I thought about doing something like the originally, but came to the conclusion that without an example of why to use this feature, people would probably just not use it.

Reply:
True, although it's arguably just as important that the fuel tanks aren't FIR than it is that they're empty. With non-FIR fuel tanks, their value is basically zero because you can't sell them.

Reply:
True. Perhaps making it say something along the lines of 'when using non-FIR fuel canisters from generator.'